mmdet/ops/dcn/src/deform_conv_cuda_kernel.cu

```diff
@@ -261,7 +262,7 @@ void deformable_im2col(
         const scalar_t *data_offset_ = data_offset.data<scalar_t>();
         scalar_t *data_col_ = data_col.data<scalar_t>();
 
-        deformable_im2col_gpu_kernel<<<GET_BLOCKS(num_kernels), CUDA_NUM_THREADS>>>(
+        deformable_im2col_gpu_kernel<<<GET_BLOCKS(num_kernels), CUDA_NUM_THREADS, 0, at::cuda::getCurrentCUDAStream()>>>(
```

Comment:
There is an inconsistency. In some kernels `at::cuda::getCurrentCUDAStream()` is specified in the execution configuration, while in others `cudaStream_t stream` is added as an argument in the function.

Reply:
Fixed. Now I use the first style - in the execution configuration.